cmd/desync/cache.go

```diff
@@ -62,7 +70,7 @@ func cache(ctx context.Context, args []string) error {
 	}
 
 	// Parse the store locations, open the stores and add a cache is requested
-	s, err := MultiStoreWithCache(n, cacheLocation, storeLocations.list...)
+	s, err := MultiStoreWithCache(n, cacheLocation, clientCert, clientKey,storeLocations.list...)
```

Comment:
Minor stylistic quibble, but it'd be consistent to have a space after the last inserted comma.